pkg/pods/pod.go

```diff
@@ -660,14 +679,24 @@ func (pod *Pod) getLaunchable(launchableStanza LaunchableStanza, runAsUser strin
 	if err != nil {
 		return nil, util.Errorf("Couldn't parse launchable location '%s' as a URL: %s", launchableStanza.Location, err)
 	}
+
+	// The path of a launchable URL is expected to end with
+	// /<launchable_id>_<version>.tar.gz. The launchable needs the
+	// <version> currently, which we parse from the URL. Future work is
+	// planned to implement more explicit versions specified in the
+	// LaunchableStanza in the pod manifest
+	version, err := versionFromLocation(locationURL)
+	if err != nil {
+		return nil, err
+	}
+
 	if launchableStanza.LaunchableType == "hoist" {
 		ret := &hoist.Launchable{
-			Location:         locationURL,
+			Version:          version,
 			Id:               launchableStanza.LaunchableId,
 			ServiceId:        serviceId,
 			RunAs:            runAsUser,
 			PodEnvDir:        pod.EnvDir(),
-			Fetcher:          uri.DefaultFetcher,
```

Comment:
the launchable doesn't need to fetch anything anymore so i'm not sure why it needs to be there